docs/RunningOnDevice.md

```diff
@@ -150,7 +150,7 @@ $ react-native run-android
 
 > Hint
 >
-> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --configuration Release`).
+> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --config Release`).
```

Comment:
Why? In the other options we do not use any abbreviation. I don't see any improvements with this change.

You also need to update it [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L245), [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L101) and [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L106).